docs/reference/query-languages/esql/esql-lookup-join.md

```diff
@@ -20,11 +20,11 @@ For example, you can use `LOOKUP JOIN` to:
 
 * Your enrichment data changes frequently
 * You want to avoid index-time processing
-* You're working with regular indices
```

Comment:
This was not entirely true, as lookup indices are still not perfectly the same as regular indices.